modules/core/src/main/java/org/apache/ignite/internal/processors/cache/transactions/IgniteTxManager.java

```diff
@@ -2959,6 +2972,8 @@ private TxRecoveryInitRunnable(ClusterNode node, MvccCoordinator mvccCrd) {
 
         /** {@inheritDoc} */
         @Override public void run() {
+            timeBag.finishGlobalStage("Started");
```

Comment:
Let's check here - is info level enabled.
Otherwise, we perform a lot of unnecessary work - finishGlobalStage includes acquiring write lock, creating some objects, etc.

Reply:
info is always enabled on adequate deployments?
useless code I think

Reply:
done